lib/iris/fileformats/netcdf.py

```diff
@@ -2563,7 +2563,12 @@ def add_ellipsoid(ellipsoid):
                     )
                     cf_var_grid.false_easting = cs.false_easting
                     cf_var_grid.false_northing = cs.false_northing
-                    cf_var_grid.scale_factor_at_projection_origin = 1.0
+                    if cs.scale_factor_at_projection_origin is not None:
+                        cf_var_grid.scale_factor_at_projection_origin = (
+                            cs.scale_factor_at_projection_origin
+                        )
+                    if cs.standard_parallel is not None:
+                        cf_var_grid.standard_parallel = cs.standard_parallel
```

Comment:
Currently neither will be set. Given that the class won't create like that, that seems ok? It would mean that someone had intentionally set both to `None`. Could stick in a default though (based on the resolution of the other comment I can add one)

Reply:
Agreed offline with @wjbenfold that this section should be rewritten to make sure that `standard_parallel` gets written and `scale_factor_at_projection_origin` stays `None` in the case that both come in populated.

Reply:
[27a3989](https://github.com/SciTools/iris/pull/4609/commits/27a3989744c97a0ab5402b671cd3f0f6dc9059a3)